lib/datadog/lambda/trace/listener.rb

```diff
@@ -48,9 +42,14 @@ def on_wrap(request_context:, cold_start:, &block)
         options[:service] = 'aws.lambda'
         options[:span_type] = 'serverless'
         Datadog::Trace.apply_datadog_trace_context(Datadog::Trace.trace_context)
-        Datadog::Trace.wrap_datadog(options) do
```

Comment:
is this replaced with the .trace('aws.lambda') call below?

Reply:
Correct